class-two-factor-core.php

```diff
@@ -987,8 +987,7 @@ public static function user_two_factor_options( $user ) {
 								<th scope="row"><input id="enabled-<?php echo esc_attr( $class ); ?>" type="checkbox" name="<?php echo esc_attr( self::ENABLED_PROVIDERS_USER_META_KEY ); ?>[]" value="<?php echo esc_attr( $class ); ?>" <?php checked( in_array( $class, $enabled_providers, true ) ); ?> /></th>
 								<th scope="row"><input type="radio" name="<?php echo esc_attr( self::PROVIDER_USER_META_KEY ); ?>" value="<?php echo esc_attr( $class ); ?>" <?php checked( $class, $primary_provider_key ); ?> /></th>
 								<td>
-									<label for="enabled-<?php echo esc_attr( $class ); ?>"><?php $object->print_label(); ?></label>
-
+									<label class="two-factor-method-label" for="enabled-<?php echo esc_attr( $class ); ?>"><?php esc_html( $object->get_label() ); ?></label>
```

Comment:
Use a class to target the label for the method selector to avoid generic `label` styles leaking. This looks good now:

<img width="956" alt="methods" src="https://user-images.githubusercontent.com/169055/194529528-252627b8-1b28-412d-adc3-96187fffa34a.png">